### DISCUSSION TITLE
Why doesn't yhat show when I use n_lags?

### DISCUSSION THREAD

**Discussion by boozesounds** [ourownstory/neural_prophet#328] [Q&A - get help using NeuralProphet]
`m = NeuralProphet(`
`    n_lags=2*12,`
`    n_forecasts=1*12,`
`    changepoints_range=0.95,`
`    n_changepoints=30,`
`    weekly_seasonality=False,`
`    batch_size=64,`
`    epochs=100,`
`    learning_rate=1.0)`
`metrics = m.fit(df, freq='5min')`
`future = m.make_future_dataframe(df, periods=1*12, n_historic_predictions=True)`
`forecast = m.predict(future)`
`fig = m.plot(forecast)`
`fig_param = m.plot_parameters()`

`print(forecast)`
`forecast = m.predict(future)`
`forecast[['ds', 'yhat1']].tail()`

`            ds                               y       yhat1  ...     ar12    trend season_daily`
`0    2021-05-01 00:00:00  2775.22  None  ...     None     None         None`
`1    2021-05-01 00:05:00  2769.96  None  ...     None     None         None`
`2    2021-05-01 00:10:00  2773.23  None  ...     None     None         None`
`3    2021-05-01 00:15:00  2773.62  None  ...     None     None         None`
`4    2021-05-01 00:20:00  2772.12  None  ...     None     None         None`
`...                  ...      ...   ...  ...      ...      ...          ...`
`8688 2021-05-31 04:00:00      NaN  None  ...  778.216  1530.59     -5.20607`
`8689 2021-05-31 04:05:00      NaN  None  ...  772.608  1530.58      -4.8789`
`8690 2021-05-31 04:10:00      NaN  None  ...  770.372  1530.57     -4.48912`
`8691 2021-05-31 04:15:00      NaN  None  ...  768.139  1530.56     -4.03877`
`8692 2021-05-31 04:20:00      NaN  None  ...   770.04  1530.55     -3.53072`

 `               ds              | yhat1`
`2021-05-31 04:00:00 | None`
`2021-05-31 04:05:00 | None`
`2021-05-31 04:10:00 | None`
`2021-05-31 04:15:00 | None`
`2021-05-31 04:20:00 | None`

**Comment by ourownstory** [comment]
@boozesounds Might it be that it is just None for the very first '2*12` rows, because those are used as inputs for the first prediction?

**Comment by suntochukwu** [comment]
Did you solve this? i'm having the exact same issue.
